VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-participant-status-list/judge-participant-status-list.component.html

```diff
@@ -128,66 +128,42 @@ <h2 class="govuk-heading-s">Participants ({{ nonJudgeParticipants.length }})</h2
-              class="govuk-body-s label-status label-status--{{ getParticipantStatusCss(participant) }}"
-            >
-              {{ getParticipantStatus(participant) }}
-            </label>
-          </div>
+          <p id="p-{{ participant.id }}-name-participant" class="govuk-heading-s govuk-!-margin-bottom-3">
+            <span *ngIf="hasInterpreterLink(participant)" class="vh-interpreter-icon">
+              <em class="far fa-2x fa-language"></em>
+            </span>
+            {{ participant.name }}
+          </p>
 
-          <div class="vh-line" *ngIf="!lastItem"></div>
-        </div>
-      </div>
-    </div>
+          <p id="p-{{ participant.id }}-hearing-role-participant" class="govuk-body-m govuk-!-margin-bottom-3">
+            {{ participant.hearing_role }}
+          </p>
 
-    <div *ngIf="!litigantInPerson" class="vh-border">
-      <div *ngFor="let participant of nonJudgeParticipants; let lastItem = last" class="govuk-grid-row govuk-!-margin-bottom-1">
-        <div id="p-row-{{ participant.id }}-participant" class="govuk-grid-column-full govuk-!-margin-top-2">
-          <!-- represented by solicitor -->
-          <div *ngIf="!litigantInPerson">
-            <p id="p-{{ participant.id }}-name-participant" class="govuk-heading-s govuk-!-margin-bottom-3">
-              {{ participant.name }}
-            </p>
-            <p id="p-{{ participant.id }}-representee" *ngIf="participant.representee" class="govuk-body-m govuk-!-margin-bottom-3">
-              {{ !isCaseTypeNone(participant) ? 'Representative' : participant.hearing_role }} for {{ participant.representee }}
-            </p>
-            <p
-              id="p-{{ participant.id }}-hearing-role-participant"
-              *ngIf="!participant.representee"
-              class="govuk-body-m govuk-!-margin-bottom-3"
-            >
-              {{ participant.hearing_role }}
-            </p>
-            <p
-              *ngIf="!isCaseTypeNone(participant)"
-              id="p-{{ participant.id }}-case-type-group-participant"
-              class="govuk-body-m govuk-!-margin-bottom-3"
-            >
-              {{ participant.case_type_group }}
-            </p>
-          </div>
+          <p id="p-{{ participant.id }}-hearing-role-participant" *ngIf="!participant.representee?isInterpreter(participant):false"
+            class="govuk-body-s govuk-!-margin-bottom-1">
+            {{ participant.hearing_role }} for <br><strong>{{ getInterpreteeName(participant.id) }}</strong>
+          </p>
+
+          <p id="p-{{ participant.id }}-representee-participant" *ngIf="participant.representee" class="govuk-body-s govuk-!-margin-bottom-1">
+            {{ isCaseTypeNone(participant) ? participant.hearing_role : 'Representative' }} for <br><strong>{{ participant.representee }}</strong>
+          </p>
+          
+          <p id="p-{{ participant.id }}-hearing-role-participant" *ngIf="!participant.representee?!isInterpreter(participant):false"
```

Comment:
just checking this is correct `?!`

Reply:
it's legal - it's a ternary statement where the first condition is to return the inverse of evaluating the `isInterpreter` function - but i would make it prettier, i think

Reply:
seems like the logic is: if the participant does not have a representee then return false if the participant is an interpreter?

Reply:
i think this would do the same
`!participant.representee && !isInterpreter(participant)`
i.e., both conditions have to be true for it to return true. participant has to _not_ have a representee and also _not_ be an interpreter. in other words: do not show if the participant has a representee or is an interpreter.

i think it can be improved further

Reply:
i suggest adding a function, displayHearingRole, which does this:

```
return !(participant.representee || isInterpreter);
```

Reply:
or maybe call it hideHearingRole and then this
```ts
return participant.representee || isInterpreter;
```

Reply:
then just invert it in the template 🤷

Reply:
I have added a showSingleHearingRole as the logic is to show the hearing role when the participant is not a representative and not an interpreter.

Reply:
Ok refactored it completely so that there is now just one hearing role element, the displayed text is determined from a function on the component based on the hearing role.  It gets rid of all the weird logic in the HTML and replaces it with a switch statement in the function

Reply:
sounds good to me, nice one @williambowshersolirius